de.dlr.sc.virsat.swtbot.test/src/de/dlr/sc/virsat/swtbot/test/ExporterImporterWizardTest.java

```diff
@@ -108,28 +98,51 @@ public void testHTMLExportFEA() {
 		assertTrue("Index file has been successfully created.", htmlExportIndexFile.exists());
 		assertTrue("ConfigurationTree file has been successfully created.", htmlExportCTFile.exists());
 		assertTrue("ElementConfiguration file has been successfully created.", htmlExportECFile.exists());
+	}
+	
+	@Test
+	public void testMatExportImport() {
+		Concept conceptMass = ConceptXmiLoader.loadConceptFromPlugin(de.dlr.sc.virsat.model.extension.budget.mass.Activator.getPluginId() + "/concept/concept.xmi");	
```

Comment:
Should we maybe load all the concepts we need in test in the base class? There is already FEA there.

Reply:
We only use the mass concept in like 1-2 places. I think I would prefer it locally loaded so we dont create additional load times in all the test cases.